docs/legacy/quickstart_legacy.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p> in how to use&nbsp;--&gt; to using</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>